app/src/main/java/org/mozilla/fenix/search/SearchFragment.kt

```diff
@@ -177,6 +189,55 @@ class SearchFragment : Fragment(), BackHandler {
             qrFeature.get()?.scan(R.id.container)
         }
 
+        val stubListener = ViewStub.OnInflateListener { _, inflated ->
+            val sharedPrefEdit = requireActivity().getSharedPreferences(
```

Comment:
It would be better to expose a setter using `var` on `Settings` instead of getting a new reference to shared preferences.